### PR TITLE
[FIX] Tags by department

### PR DESCRIPTION
**Pull request by KevLehman**
## Further comments

EOC-51

### CODE REVIEW

**Review by MartinSchoeler — DISMISSED** [review]
It still seems that any agent regardless of department can access all tags when closing a chat. Maybe there is still something missing?

**Comment by KevLehman** [comment]
Thanks @MartinSchoeler , let me confirm :)

File: client/components/Omnichannel/modals/CloseChatModal.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ const CloseChatModal = ({ department = {}, onCancel, onConfirm }) => {
 				</Field>
 				{Tags && (
 					<Field>
-						<Tags tagRequired={tagRequired} tags={tags} handler={handleTags} error={tagError} />
+						<Tags tagRequired={tagRequired} handler={handleTags} department={department} />
```
Comment (MartinSchoeler) [inline client/components/Omnichannel/modals/CloseChatModal.js:88]: The issue seems to be that department always = {}

**Comment by murtaza98** [comment]
Closing this in favour of https://github.com/RocketChat/Rocket.Chat/pull/29169

**Pull request closed without merging**